src/main/java/com/training/library/service/DatabaseService.java

```diff
@@ -0,0 +1,95 @@
+package com.training.library.service;
+
+import com.training.library.entity.Book;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DatabaseService {
+
+    private final String url = "jdbc:mysql://localhost:3306/library?serverTimezone=Europe/Moscow&useSSL=false";
```

Comment:
All fields here can be static. Also you can create a Properties manager and create a properties file which will contain this values. You can read about property files [here](https://devcolibri.com/%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%BE%D0%B9-%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%80-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%8B-%D1%81-property-%D1%84%D0%B0%D0%B9%D0%BB%D0%B0%D0%BC%D0%B8-%D0%B2-java/).